eppo_client/rules.py

```diff
@@ -119,3 +119,13 @@ def compare_semver(
         return semver.compare(attribute_value, condition_value) <= 0
 
     return False
+
+
+def to_string(value: AttributeType) -> str:
```

Comment:
Handle an array of strings?

Reply:
null?

Reply:
You could normalize boolean, `None` and arrays by converting to JSON. Just about every language has some library that can convert to JSON, so this would work for every SDK:
<img width="259" alt="Screenshot 2024-06-07 at 8 25 54 AM" src="https://github.com/Eppo-exp/python-sdk/assets/1248641/9e345bb5-8d2f-462c-881d-7af22c055858">

So the logic might be:

`json.dumps(int(value)) if value.is_integer() else json.dumps(value)`

But you'd also need to parse the json for the configuration value to compare for equality

Reply:
`AttributeType = Union[str, int, float, bool]`

Attribute cannot be an array -- but good call that we do want to handle null. 

Using `json` is a great suggestion, but I do want to make sure we aren't losing a lot of performance (I've seen `json` being very slow, but perhaps here it's sufficiently quickly given that the input is small)

Reply:
Yeah, that's a valid concern. It still might be useful as an "else" case for any value that we don't expect, while handling the cases that we expect in a more performant way, but I'm good with whatever you decide so long as we handle `None` / `null`

Reply:
![Screenshot 2024-06-07 at 10 30 35 AM](https://github.com/Eppo-exp/python-sdk/assets/3537793/9d76f461-3e99-4041-b2f0-b5d2dcce6490)

Seems like using json is equally fast

Reply:
it looks like they're equal because they both landed on `if isninstance(value, str):`. You should try it out with no conditionals -- just running the functions we want tested: (`str` vs `json.dump`)

Reply:
But in any case, the `json.dump` only happens after all other conditionals fail, so we're still good here.

Reply:
Oh good catch 😓